### PR TITLE
Sapi

### PR DESCRIPTION
**Pull request by sapikara**
Updating README.md and setup.py and adding requirement.py.

### CODE REVIEW

**Review by andy22b — COMMENTED** [review]
Looks good. I'm not sure we need the xml files included? They are PyCharm specific...

File: README.md [hunk 1]
```diff
@@ -1,7 +1,57 @@
...
+* python >=Python 3.7
+
+* python pip3
+
+* python virtualenv
+
+##### 3. Create and activate Python virtualenv
+
+````
+mkdir eq-fault-geom/ENV
```
Comment (chrisbc) [inline README.md:26]: IN hindsight these two lines are best left out. I will remove them from the other project too.

**Review by chrisbc — CHANGES_REQUESTED** [review]
Hi Sapi, 

 - I like the info about how setup.py and requirements.txt are different. 
 - I usually run the tests like nosetests, but this seems more canonical
 - I made a comment about some lines added into readme. 
 - adding the .idea folder hmmm... I don't think this should be part of the project, as it's distracting for non idea users. And who knows -  your settings may conflict with other .idea users?
 - there is already a .gitignore file in the project 

Can you make your changes on the sapi branch and request another review please. Then I'll 
checkout your work and try it out :)

**Review by chrisbc — APPROVED** [review]
:+1:

**Review by chrisbc — APPROVED** [review]
:+1:

**Pull request merged**